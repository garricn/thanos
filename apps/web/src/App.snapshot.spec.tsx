import { render } from '@testing-library/react';
import App from './App';

describe('App Snapshots', () => {
  it('should render correctly', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
