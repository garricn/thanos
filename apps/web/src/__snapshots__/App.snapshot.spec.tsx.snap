// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Snapshots > should match snapshot 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
